### PR TITLE
feat(console): revamp map view

### LINKED ISSUE #6455: Some issues with new map view
### I tried this:

https://github.com/winglang/containers-example/

### This happened:

This was the result:

image

Previously it looked like this:

image

### I expected this:

Issues:
1. We are missing *some* manually created connections

2. No "access" information on connections

3. Some layout issue with the connection from the queue:
   image

4. It appears like `forward()` is an inflight method of `Queue` and it is not and `database()` is an inflight method of `Workload`. They are manually created connections, not methods. Ideally I would love the text "forward" and "database" to appear on the connection line itself and not as a method.

5. The lines are not centered vertically:
   image

### Is there a workaround?

_No response_

### Anything else?

_No response_

### Wing Version

_No response_

### Node.js Version

_No response_

### Platform(s)

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### LINKED ISSUE #6459: custom class internals are not rendered correctly
### I tried this:

Start with this code:
```wing
bring cloud;
class X {
  b: cloud.Bucket;
  new(){
    this.b = new cloud.Bucket();
  }
  pub f(){
    this.b.onEvent(inflight () => {});
  }
  pub inflight ff(){
    this.b.put("sdads","dsadsa");
  }
}
let api = new cloud.Api();
let b = new cloud.Bucket();
new cloud.Function(inflight () => {
  b.put("file.txt", "dsasd" );
  b.get("dsasd");
});
new cloud.Function(inflight () => {
  b.put("file.txt", "dsasd" );
  b.get("dsasd");
  b.list();
}) as "2";
api.post("/doc/:id", inflight (req) => {
  
});
```
It should render something like this:
image

Now add `let x = new X();` at the end of the file, it renders this:
image

### This happened:

If I stop and start `wing run` it will show a correct representation: 
image

### I expected this:

It should have generate the right structure 

### Is there a workaround?

_No response_

### Anything else?

_No response_

### Wing Version

0.73.41

### Node.js Version

_No response_

### Platform(s)

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### LINKED ISSUE #6511: Console arrows are hidden on dark mode
### I tried this:

https://github.com/winglang/wing/assets/5008987/edc190da-30f5-47ce-82ca-8bddc433f7f3

### This happened:

The arrows blend in with the background, so it looks like there are no connections

### I expected this:

_No response_

### Is there a workaround?

Switch to light mode

### Anything else?

_No response_

### Wing Version

0.73.51

### Node.js Version

_No response_

### Platform(s)

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### PR DESCRIPTION
**Pull request by skyrpex**
This restores the new Console map view again, and will try to fix the previous issues with the missing connections.

Fixes #6511, fixes #6459, fixes #6455.

### CODE REVIEW

**Comment by eladb** [comment]
@skyrpex can you please share a screenshot of how [this app](https://github.com/winglang/containers-example/) looks like now?

File: apps/wing-console/console/design-system/src/row-input.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const RowInput = memo(
           type === "checkbox" && [
             theme.focusInput,
             theme.bg4,
-            "w-4 h-4 dark:ring-offset-gray-800",
+            "w-4 h-4 dark:ring-offset-slate-800",
```
Comment (eladb) [inline apps/wing-console/console/design-system/src/row-input.tsx:71]: image


Still some issues with dark mode

**Comment by eladb** [comment]
> @skyrpex can you please share a screenshot of how [this app](https://github.com/winglang/containers-example/) looks like now?

⬆️

**Comment by skyrpex** [comment]
@eladb 
image

File: apps/wing-console/console/design-system/src/row-input.tsx (hunk 1 above)
Comment (skyrpex) [reply apps/wing-console/console/design-system/src/row-input.tsx:71]: Fixing today

**Review by eladb — APPROVED** [review]

**Comment by eladb** [comment]
Ship it!

**Review by polamoros — APPROVED** [review]

File: apps/wing-console/console/design-system/src/row-input.tsx (hunk 1 above)
Comment (skyrpex) [reply apps/wing-console/console/design-system/src/row-input.tsx:71]: Fixed.

**Pull request merged**